Harold Farmer and Stan Hostetler Changes

haroldfarmer opened the pull request:
User functionality improved by implementation of GUI. Will continue to work on improvements in future projects.

haroldfarmer committed:
add parts of gui to make more user friendly

mikeknows commented:
Merge conflict with the imports after merging the logging class. Please fix imports and I will merge.

Thanks

haroldfarmer committed:
Merge branch 'master' into gui

haroldfarmer commented:
Fixed the imports

haroldfarmer committed:
Added fixes

I looked at the travis ci because it failed so I added some fixes. When I tried to fix the merge conflicts I made a few mistakes.

mikeknows commented on src/wrestlingtournamentcli/Main.java:
```diff
@@ -1,139 +1,355 @@
-                + "VIEW-WRESTLER WrestlerName //Looks for the wrestler and prints his/her information\n"
-                + "UPDATE-MATCH matchNumber winningColor greenPoints redPoints fallType(int) fallTime\n");
-                }
-}
```
Need to be able to see the changes made.

mikeknows commented on .classpath:
```diff
@@ -1,6 +1,11 @@
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/">
```
DIscard or explain changes to .classpath

mikeknows reviewed (commented):
What are the filehashes.bin and filehases.lock files for?

haroldfarmer commented:
Do you still have office hours or a discord? Im a little confused on why .classpath is changed and why filehashes.bin and filehases.lock is showing up. We didn't touch or create any of these files. I also am a little confused on how git works.

mikeknows commented:
The computer science dept. has made a discord server with 3120 https://discord.gg/ks3pV7G

Also there is a slack server: raiderplanner.slack.com

I'll be available in both.

haroldfarmer commented:
Will it be the same time as your office hours?

mikeknows commented:
> 
> 
> Will it be the same time as you office ours?

Yes and by appointment
